flake.nix

```diff
@@ -325,7 +325,7 @@
           checks = {
             cargo-fmt = pkgs.runCommand "check-cargo-fmt"
               { inherit (self.devShells.${system}.default) nativeBuildInputs buildInputs; } ''
-              cargo fmt --manifest-path=${./.}/Cargo.toml --check > $out
+              cargo fmt --manifest-path=${./.}/Cargo.toml --check && touch $out
```

Comment:
I think you might be looking for this: https://stackoverflow.com/questions/418896/how-to-redirect-output-to-a-file-and-stdout

Reply:
I don't think we need the output in the file…what do you say @wucke13